src/main/webapp/app/exam/manage/students/exam-students.component.ts

```diff
@@ -74,7 +79,15 @@ export class ExamStudentsComponent implements OnInit, OnDestroy {
         this.isAdmin = this.accountService.isAdmin();
         this.route.data.subscribe(({ exam }: { exam: Exam }) => {
             this.exam = exam;
-            this.allRegisteredUsers = exam.registeredUsers! || [];
+            this.allRegisteredUsers =
+                (exam.examUsers &&
+                    exam.examUsers.map((examUser) => {
```

Comment:
You can probably use optional chaining here, to make it more readable, i.e `exam.examUsers?.map...`